hub/powertoys/run.md

```diff
@@ -106,13 +106,13 @@ PowerToys Run enables a set of system level actions that can be executed.
 
 | Action command / Search result | Action | Note |
 | :--- | :--- | :--- |
-| `Shutdown` | Shuts down the computer | |
-| `Restart` | Restarts the computer | |
-| `Sign Out` | Signs current user out | |
-| `Lock` | Locks the computer | |
-| `Sleep` | Sleeps the computer | |
-| `Hibernate` | Hibernates the computer | |
-| `Empty Recycle Bin` | Empties the recycle bin | |
+| `Shutdown` | Shuts down the computer | Can use localised keyword |
+| `Restart` | Restarts the computer | Can use localised keyword |
+| `Sign Out` | Signs current user out | Can use localised keyword |
+| `Lock` | Locks the computer | Can use localised keyword |
+| `Sleep` | Sleeps the computer | Can use localised keyword |
+| `Hibernate` | Hibernates the computer | Can use localised keyword |
+| `Empty Recycle Bin` | Empties the recycle bin | Can use localised keywords |
```

Comment:
good idea. but not a good way to implement.

Reply:
Well, I added it because (for me) the English words don't work at all.

Reply:
sure. but there is a plugin setting.